src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java

```diff
@@ -9,15 +9,16 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class INewsService implements NewsService {
+public class NewsServiceImpl implements NewsService {
+    @Autowired
     private NewsMapper newsMapper;
     @Autowired
     private NewsRepository newsRepo;
 
     @Override
     public NewsDTO getDetailsById(String id) {
         System.out.println(id);
```

Comment:
What is the function of this line?

Reply:
I try to refactor the name of the class, buy in my new PR I finally deleted this and didn't implement the service, so that the mapstruct dependency could work

Reply:
right I forgot to erase it! thanks a ton!